Makefile

```diff
@@ -78,3 +78,7 @@ curl_local curl_prod:
 		"N":["C7", "H6", "H7", "H9", "CJ", "SA", "S8", "SQ", "D5", "S5", "HK", "C8", "HA"],    \
 		"E":["H2", "H5", "CQ", "D9", "H4", "ST", "HQ", "SJ", "HJ", "DQ", "H3", "C9", "CK"] }}' \
 	${CURL_URL}
+
+start_local_server: libdds-build
+	pip3 install -q -r requirements.txt
+	python3 -m src.api
```

Comment:
Would we want to start the service in background? We could have something like:

```
.build/service-pid: libdds-build .build/requirements-stamp stop_local_service
    @echo "Starting a local test service as a background process. You can stop it with make stop_local_service"
    pythos -m src.api & echo $? > $@

stop_local_service:
   [ ! -e .build/service-pid ] || (kill $(cat .build/service-pid); rm .build/service-pid)

local_curl: .build/service-pid
```

Reply:
I'll add that.

Shouldn't the output be redirected, say to `api.log` or `server.log`?

I presume that 'pythos' is a typo.

Reply:
Yes. Typo. There is other typos like missing second dollar for the subshell syntax.

Yeah. Output probably should redirected. Maybe inside the .build directory.

Reply:
How's this?

```
logfile := logs/src.api.$(shell date +%FT%T).log
pidfile := server.PID

start_local_server: libdds-build stop_local_server
	python3 -m src.api > ${logfile} 2>&1 & echo $$! > ${pidfile}

stop_local_server:
	[ ! -e ${pidfile} ] || (kill $$(cat ${pidfile}) && rm ${pidfile})
```

Reply:
Looks better than my original rough idea. But I would but server.PID inside a directory like logs or build. I like to have my source directories clean from build output. Excluding files which are supposed to be packaged in a source archive like a file simultating git describe for a source archive.

Reply:
Done.